keras/models.py

```diff
@@ -976,19 +976,37 @@ def evaluate(self, x=None, y=None,
                  verbose=1,
                  sample_weight=None,
                  steps=None):
-        """Computes the loss on some input data, batch by batch.
+        """Returns the loss value & metrics values for the model in test mode.
+
+        Computation is done in batches.
 
         # Arguments
-            x: input data, as a Numpy array or list of Numpy arrays
-                (if the model has multiple inputs).
+            x: Numpy array of input data.
+                If the input layer in the model is named, you can also pass a
+                dictionary mapping the input name to a Numpy array.
```

Comment:
I think for `Sequential` we shouldn't mention dictionaries, since it is a worse API in the single-input, single-output model. It will just confuse users

Reply:
removed